code/game/objects/items.dm

```diff
@@ -420,7 +420,9 @@
 
 	if(istype(H))
 		var/obj/item/organ/eyes/eyes = H.internal_organs_by_name["eyes"]
-		if(!eyes)
+		if(!eyes) // should still get stabbed in the head
+			var/obj/item/organ/external/head/head = H.organs_by_name["head"]
+			head.take_damage(rand(2,3), 1) //being stabbed in the head isnt as bad as the eyes

```

Comment:
Considering that people with eyes take 3-4 eye damage and 7 damage to the head and have a chance to be weakened, it may be best to make this rand(10,14)